src/medCore/medPlugin.h

```diff
@@ -0,0 +1,96 @@
+/*=========================================================================
+
+ medInria
+
+ Copyright (c) INRIA 2013 - 2014. All rights reserved.
+ See LICENSE.txt for details.
+
+  This software is distributed WITHOUT ANY WARRANTY; without even
+  the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+  PURPOSE.
+
+=========================================================================*/
+
+#pragma once
+
+#include <dtkCore/dtkPlugin.h>
+#include <dtkLog/dtkLog.h>
+#include <medCoreExport.h>
+
+class MEDCORE_EXPORT medPlugin : public dtkPlugin
+{
+    Q_OBJECT
+    Q_INTERFACES(dtkPlugin)
```

Comment:
Q_INTERFACES has no use here and should not be removed from each of the derived plugin classes. This macro is meant to be used in plugins to declare that a particular class inherits from an interface declared in the base application. The proper use would be to put Q_DECLARE_INTERFACE in medPlugin, and Q_INTERFACES(medPlugin) in the derived classes.